addons/sale_timesheet/controllers/portal.py

```diff
@@ -132,7 +132,3 @@ def _task_get_page_view_values(self, task, access_token, **kwargs):
                     'title': title,
                 })
         return values
-
-    @http.route(['/my/timesheets', '/my/timesheets/page/<int:page>'], type='http', auth="user", website=True)
-    def portal_my_timesheets(self, page=1, sortby=None, filterby=None, search=None, search_in='all', groupby='sol', **kw):
-        return super().portal_my_timesheets(page, sortby, filterby, search, search_in, groupby, **kw)
```

Comment:
```suggestion
    @http.route()
    def portal_my_timesheets(self, page=1, sortby=None, filterby=None, search=None, search_in='all', groupby='sol', **kw):
        return super().portal_my_timesheets(page, sortby, filterby, search, search_in, groupby, **kw)
```
In fact, we change the signature to change the default value of `groupby`. We want group by SOL by default when `sale_timesheet` module is installed. 😇

Reply:
Indeed, missed the groupby change :man_facepalming:
```python

    @http.route()
    def portal_my_timesheets(self, *args, groupby='sol', **kw):
        return super().portal_my_timesheets(*args, groupby=groupby, **kw)
```
then ? (would make it clearer that only the default groupby is modified)

Reply:
Why not, I agree. 🙂
Thanks! 🙂